### PR TITLE
[workers] routes revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: products/workers/src/content/platform/routes.md [hunk 1]
```diff
@@ -13,25 +13,25 @@ Routes allow users to map a URL pattern to a Worker script to enable Workers to
...
+- Route URL (refer to [Matching Behavior](#matching-behavior))
 - Worker script to execute on matching requests
-- Failure mode for rate-limited accounts on the free plan (see [daily request limits](/platform/limits#request-limits))
+- Failure mode for rate-limited accounts on the Free plan (see [daily request limits](/platform/limits#request-limits))
 
-The Routes REST API documentation can be found [in the Workers API docs](https://api.cloudflare.com/#worker-routes-properties).
+The Routes REST API documentation can be found [in the Workers API documentation](https://api.cloudflare.com/#worker-routes-properties).
 
-\* _A zone that you have registered with some registrar (not workers.dev) and setup Cloudflare to serve as [a reverse proxy](https://www.cloudflare.com/learning/cdn/glossary/reverse-proxy/)._
+\* A zone that you have registered with some registrar (not a `*.workers.dev` subdomain) and set up Cloudflare to serve as [a reverse proxy](https://www.cloudflare.com/learning/cdn/glossary/reverse-proxy/).
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/routes.md]: @lukeed I feel like we left this in here for a good reason but I am now confused at how this sentence is floating with the \* in the beginning. Is this supposed to match with line 14?

Comment (lukeed) [reply products/workers/src/content/platform/routes.md]: I don't think I've ever seen this – if we did then the workers.dev would have already been formatted

File: products/workers/src/content/platform/routes.md [hunk 2]
```diff
@@ -13,25 +13,25 @@ Routes allow users to map a URL pattern to a Worker script to enable Workers to
...
```
Comment (deadlypants1973) [inline products/workers/src/content/platform/routes.md]:
```suggestion
```

File: products/workers/src/content/platform/routes.md [hunk 3]
```diff
@@ -13,25 +13,23 @@ Routes allow users to map a URL pattern to a Worker script to enable Workers to
 
 For zones proxied on Cloudflare\*, route patterns decide what (if any) script is matched based on the URL of that request. Requests are routed through a Workers script when the URL matches a route pattern assigned to that script.
 
-Route patterns can be added with the Cloudflare API or on the [Workers tab](https://dash.cloudflare.com/?zone=workers) in the Cloudflare dashboard (after selecting a zone).
+Route patterns can be added with the Cloudflare API or in **Account Home** > [**Workers**](https://dash.cloudflare.com/?zone=workers) > **your Worker** > **Triggers** > **Add route** in the Cloudflare dashboard.
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md:16]: is `your` in `your Worker` lowercase in the interface?

File: products/workers/src/content/platform/routes.md [hunk 4]
```diff
@@ -54,7 +52,7 @@ While they look similar to a [regex](https://en.wikipedia.org/wiki/Regular_expre
 
 - The only supported operator is wildcard `*` which matches zero or more of any character.
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md]:
```suggestion
- The only supported operator is the wildcard (`*`), which matches zero or more of any character.
```

File: products/workers/src/content/platform/routes.md [hunk 5]
```diff
@@ -54,7 +52,7 @@ While they look similar to a [regex](https://en.wikipedia.org/wiki/Regular_expre
...
 
-- Route patterns may not contain infix wildcards or query parameters, e.g. neither `example.com/*.jpg` nor `example.com/?foo=*` are valid route patterns.
+- Route patterns may not contain infix wildcards or query parameters, for example, neither `example.com/*.jpg` nor `example.com/?foo=*` are valid route patterns.
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md]:
```suggestion
- Route patterns may not contain infix wildcards or query parameters. For example, neither `example.com/*.jpg` nor `example.com/?foo=*` are valid route patterns.
```

File: products/workers/src/content/platform/routes.md [hunk 6]
```diff
@@ -73,46 +71,46 @@ In this example, all requests destined for example.com and whose paths are prefi
 
 ### Validity
 
-Here is the full set of rules governing route pattern validity:
+The set of rules governing route pattern validity:
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md]: should this be a complete sentence? e.g., `Route pattern validity is governed by the following set of rules:` (excuse the passive)

File: products/workers/src/content/platform/routes.md [hunk 7]
```diff
@@ -73,46 +71,46 @@ In this example, all requests destined for example.com and whose paths are prefi
...
-- `https://*.example.com/` matches `https://www.example.com/` but _not_ `http://www.example.com/`
+- `https://*.example.com/` matches `https://www.example.com/` but not `http://www.example.com/`.
 
-- `*.example.com/` matches both `https://www.example.com/` _and_ `http://www.example.com/`.
+- `*.example.com/` matches both `https://www.example.com/` and `http://www.example.com/`.
 
 #### Hostnames may optionally begin with `*`
 
-If a route pattern hostname begins with `*`, then it matches the host *and* all subhosts. If a route pattern hostname begins with `*.`, then it matches *only* all subhosts.
+If a route pattern hostname begins with `*`, then it matches the host and all subhosts. If a route pattern hostname begins with `*.`, then it matches only all subhosts.
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md]:
```suggestion
If a route pattern hostname begins with `*`, then it matches the host and all subhosts. If a route pattern hostname begins with `*.`, then it only matches all subhosts.
```

File: products/workers/src/content/platform/routes.md [hunk 8]
```diff
@@ -73,46 +71,46 @@ In this example, all requests destined for example.com and whose paths are prefi
...
 
 If a route pattern path ends with `*`, then it matches all suffixes of that path.
 
-- `https://example.com/path*` matches `https://example.com/path` _and_ `https://example.com/path2` _and_ `https://example.com/path/readme.txt`
+- `https://example.com/path*` matches `https://example.com/path` and `https://example.com/path2` and `https://example.com/path/readme.txt`
 
 <Aside type="warning">
 
-There is a well-known bug associated with path matching concerning wildcards and forward slashes that is documented in the [Known issues section.](/platform/known-issues)
+There is a well-known bug associated with path matching concerning wildcards (`*`) and forward slashes (`/`) that is documented in [Known issues.](/platform/known-issues)
```
Comment (abracchi-tw) [inline products/workers/src/content/platform/routes.md]:
```suggestion
There is a well-known bug associated with path matching concerning wildcards (`*`) and forward slashes (`/`) that is documented in [Known issues](/platform/known-issues).
```

**Review by abracchi-tw — APPROVED** [review]

File: products/workers/src/content/platform/routes.md (hunk 6 above)
Comment (deadlypants1973) [inline products/workers/src/content/platform/routes.md]:
```suggestion
The following set of rules govern route pattern validity.
```

**Pull request merged**